tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskRestBasicUsageTests.scala

```diff
@@ -450,12 +456,26 @@ class WskRestBasicUsageTests extends TestHelpers with WskTestHelpers with WskAct
       }
 
       val action = wsk.action.get(name)
-      action.getFieldJsValue("annotations") shouldBe JsArray(
-        JsObject("key" -> JsString("web-export"), "value" -> JsBoolean(true)),
-        JsObject("key" -> JsString("raw-http"), "value" -> JsBoolean(false)),
-        JsObject("key" -> JsString("final"), "value" -> JsBoolean(true)),
-        JsObject("key" -> WhiskAction.provideApiKeyAnnotationName.toJson, "value" -> JsBoolean(false)),
-        JsObject("key" -> JsString("exec"), "value" -> JsString("nodejs:6")))
+      action.getFieldJsValue("annotations") shouldBe (if (requireAPIKeyAnnotation) {
+                                                        JsArray(
+                                                          JsObject("key" -> JsString("web-export"), "value" -> JsTrue),
+                                                          JsObject("key" -> JsString("raw-http"), "value" -> JsFalse),
+                                                          JsObject("key" -> JsString("final"), "value" -> JsTrue),
+                                                          JsObject(
+                                                            "key" -> WhiskAction.provideApiKeyAnnotationName.toJson,
+                                                            "value" -> JsFalse),
+                                                          JsObject(
+                                                            "key" -> JsString("exec"),
+                                                            "value" -> JsString("nodejs:6")))
+                                                      } else {
+                                                        JsArray(
+                                                          JsObject("key" -> JsString("web-export"), "value" -> JsTrue),
+                                                          JsObject("key" -> JsString("raw-http"), "value" -> JsFalse),
+                                                          JsObject("key" -> JsString("final"), "value" -> JsTrue),
+                                                          JsObject(
+                                                            "key" -> JsString("exec"),
+                                                            "value" -> JsString("nodejs:6")))
+                                                      })
```

Comment:
Please adjust the way the annotations are built to the same method as used above.

Reply:
Changed as asked. To the price that I had to revert back to the `JSBoolean()` value format to make it compile.